integration-tests/src/test/scala/eu/neverblink/jelly/integration_tests/rdf/helpers/TitaniumDatasetEmitter.scala

```diff
@@ -25,7 +25,23 @@ object TitaniumDatasetEmitter:
       else
         val o = transformResource(q.getObject.asInstanceOf[RdfResource])
         output.quad(s, p, o, null, null, null, g)
+  
+  def emitDatasetTo(ds: Seq[RdfNQuad], output: RdfQuadConsumer): Unit =
+    for q <- ds do
```

Comment:
this is duplicated code from the above – if this is needed, please factor this out somehow.